frontend/layouts/discover-page/component.tsx

```diff
@@ -120,24 +120,27 @@ export const DiscoverPageLayout: FC<DiscoverPageLayoutProps> = ({
 
   return (
     <div className="fixed top-0 bottom-0 left-0 right-0 h-screen bg-background-dark">
-      <div className="flex flex-col h-screen overflow-auto">
+      <div className="flex flex-col h-screen sm:overflow-auto bg-radial-green-dark bg-[length:100%_166px] bg-no-repeat sm:bg-none">
```

Comment:
Can we use the same background colour as the desktop instead for the sake of consistency (`bg-green-dark`)?